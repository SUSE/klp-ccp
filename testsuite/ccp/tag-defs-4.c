enum A
{
	a_a,
};

void pu_f(enum A *a)
{
	*a;
}
