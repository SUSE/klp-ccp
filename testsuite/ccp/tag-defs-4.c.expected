enum A
{
	a_a,
};

void klpp_pu_f(enum A *a)
{
	*a;
}
